swan-lake/featured-scenarios/work-with-data-using-queries-in-ballerina.md

```diff
@@ -426,31 +426,35 @@ Countries with highest deaths:[["India",980976],["USA",880976],["Afghanistan",73
 Countries with number of Recovered patients:[["Afghanistan",146084],["USA",43892277],["India",33892279]]
 Found erroneous entries for countries: ["Sri Lanka","India"]
 ```
-## Group COVID-19 data by continent
 
-### Create the `groupDeathsByContinent` function
+## Aggregating
+
+### Create the `getTotalNumberOfDeaths` function
+
+In order to aggregate the input records of query we can use `collect` clause. If we use `collect` clause, all the variables in the query expression become aggregated variables. We can use aggregated variable as a rest argument to a function call or single element list constructor. 
+
+We can only use langlib functions with aggregated variables. Also, it is not necessary to use module prefix in langlib function call. For example, we can write `sum(deaths)` instead of `int:sum(deaths)`. Basic aggregating functions such as `min`, `max`, `avg`, `sum`, `first`, `last`, and `count` are available.
```

Comment:
rather saying simply `use` shall we explain it well? e.g. aggregated variables/sequence variables can only be used as an argument for rest parameters of the langlib function.

Reply:
Changed